src/features/Applet/Schedule/Calendar/Calendar.types.ts

```diff
@@ -0,0 +1,21 @@
+export type CalendarEvent = {
+  title: string;
+  start: Date;
+  end: Date;
+  id: string;
+  backgroundColor: string;
+  alwaysAvailable: boolean;
+  startIndicator?: string;
+  startTime?: string;
+  startFlowIcon?: boolean;
+  endAlertIcon?: boolean;
+  isHidden?: boolean;
+  isOffRange?: boolean;
+};
+
+export const enum CalendarView {
+  day = 'day',
```

Comment:
Let's use enums with capitalize 
export const enum CalendarView {
  Day

Reply:
React big calendar uses not capitalize views.
![image](https://user-images.githubusercontent.com/116339462/215470384-a864065b-35a8-4003-b2ec-65a9d49bb4dd.png)